spec/Classes.tex

```diff
@@ -309,11 +309,14 @@ \subsection{Overriding Base Class Methods}
 dispatch in the event that a variable that has the base class type
```

Comment:
You've inherited all of this, but since I'm reading it:

"the base class type" -> "a base class type"?  (or "...parent..." or "...ancestor..."?).  I'm on the fence about this.  On one hand, the previous text has restricted our scope from all base classes to a particular one; but on the other hand, this statement applies if the variable has any of the base class types.

Should this refer to "static type" in some way for improved clarity?  ("a variable whose static type is a base class type...")